ndk/src/native_window.rs

```diff
@@ -37,14 +37,6 @@ impl Clone for NativeWindow {
     }
 }
 
-unsafe impl HasRawWindowHandle for NativeWindow {
```

Comment:
Why/when has the trait been removed?

Reply:
https://github.com/rust-windowing/raw-window-handle/blob/0ee134f812cc7191cc2ee33a1fdf2d266f0cf1e4/src/lib.rs#L60 It's still there on master?

Reply:
https://github.com/rust-windowing/raw-window-handle/blob/master/CHANGELOG.md#050-2022-07-14 This is not in the changelog for `0.5.0` either.

Reply:
This trait is a compatibility for raw window handle `v0.3`, there's no need for it with `v0.5.0` it was left to simplify transition, but there's no need in it, since the handle is cross platform.

Reply:
Hm, I see. Will update.